inc/cropper/namespace.php

```diff
@@ -124,7 +124,7 @@ function rest_api_fields( WP_REST_Response $response ) : WP_REST_Response {
 	}
 
 	// Confirm it's definitely an image.
-	if ( ! isset( $data['id'] ) || ! isset( $data['media_type'] ) ) {
+	if ( ! isset( $data['id'] ) || ! isset( $data['media_type'] ) || ! is_array( $data['media_details'] ) ) {
```

Comment:
Might be better to just check this on line 144 - would need to confirm that SVGs have the media details first.